.github/workflows/ci.yaml

```diff
@@ -129,3 +129,19 @@ jobs:
 
       - name: Clippy check
         run: cargo clippy --features=linux-native -- -D warnings
+
+  doc_build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Fetch head
+        uses: actions/checkout@v4
+
+      - name: Install rust stable
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: nightly
+          targets: [x86_64-unknown-linux-gnu, aarch64-apple-darwin, aarch64-apple-ios, x86_64-pc-windows-msvc]
```

Comment:
I am not sure about the syntax for setting multiple targets. Since every change and test require approve to trigger the CI, @brotskydotcom please fill free to directly add commit and change this.

Reply:
Nevermind, I [test in my own crate](https://github.com/unkcpz/rsdos/actions/runs/13592911412/workflow), and works now with comma split target names. See the last commit.